### ISSUE TITLE
Ability to edit colors in colorsbook

### ISSUE THREAD

**Issue by angelo** [Toinane/colorpicker#24]
![image](https://user-images.githubusercontent.com/488/38167254-2d112636-3500-11e8-8dfa-c1231c564784.png)

It'd be very useful to be able to edit these colors once defined here. Also, perhaps adding rgba as an option would be handy, as opacity can be defined in the picker.

**Comment by rodrigorhas** [comment]
@Toinane Maybe if we can describe the color with labels, comments or even tags. This will be a lot useful for me and others

**Comment by Toinane** [comment]
Yes I think too. In any case it was planned to upgrade the colorsbook. for the moment, I've quickly implemented colorsbook to be able to save colors easily but this version isn't really UX and I don't like it.

In my roadmap, the next version will be a refactoring of all code to typescript and I will make a new version just for Linux to enable the picker feature finally!
Afterward I'll take care of the colorsbook!

**Comment by Toinane** [comment]
@rodrigorhas You can ignore this branch, it was a test :) I've created a new branch 'refacto-typescript' #36 
I'm on it actually! I rewrite all the code (old code is in 'old' folder)
